Sources/TuistCache/Utilities/CacheXCFrameworkBuilder.swift

```diff
@@ -87,19 +85,17 @@ public final class CacheXCFrameworkBuilder: CacheArtifactBuilding {
 
     // MARK: - Fileprivate
 
-    fileprivate func buildXCFramework(frameworks: [AbsolutePath], output: AbsolutePath) throws {
-        _ = try xcodeBuildController.createXCFramework(frameworks: frameworks, output: output)
-            .toBlocking()
-            .last()
+    fileprivate func buildXCFramework(frameworks: [AbsolutePath], output: AbsolutePath) async throws {
+        try await xcodeBuildController.createXCFramework(frameworks: frameworks, output: output).printFormattedOutput()
     }
```

Comment:
nit: I think we can just remove this function and directly use xcodeBuildController above now that's much simpler